mcp-server/docs/WORKFLOW-CONFIG-EXAMPLE.md

```
@@ -0,0 +1,41 @@
+```yaml
+project:
+  name: my-project
+  
+features:
+  workflow:
+    autoCommit: false  # Set to false to enable manual testing checkpoint
+```
+
+## Behavior
+
+### When `autoCommit: false` (or not set - default behavior)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)